Tests/FoundationTests/FileManagerExtensionsTests.swift

```diff
@@ -80,6 +80,31 @@ final class FileManagerExtensionsTests: XCTestCase {
         } catch {}
     }
 
+    func testDecode() {
+        let point = CGPoint(x: 1, y: 2) //CGPoint conforms to Encodable
+        do {
+            let temporaryDirectoryURL = FileManager.default.temporaryDirectory
+            let temporaryFilename = ProcessInfo().globallyUniqueString
+            let temporaryFileURL = temporaryDirectoryURL.appendingPathComponent(temporaryFilename)
+
+            let encodedPoint = try JSONEncoder().encode(point)
+            XCTAssertTrue(FileManager.default.createFile(atPath: temporaryFileURL.path, contents: encodedPoint, attributes: nil))
```

Comment:
No need fro `True`
```suggestion
            XCTAssert(FileManager.default.createFile(atPath: temporaryFileURL.path, contents: encodedPoint, attributes: nil))
```

Reply:
@guykogus applied changes to all comments